test/pylib/minio_server.py

```diff
@@ -206,6 +206,13 @@ def _get_environs(self):
                 self.ENV_ACCESS_KEY,
                 self.ENV_SECRET_KEY]
 
+    def get_envs_settings(self):
+        environs = {}
+        for key in self._get_environs():
+            value = os.environ[key]
+            environs[key]=value
+        return environs
```

Comment:
Wdyt about replacing it with dict comprehension?

```python
return {key: os.environ[key] for key in self._get_environs()}
```

And what if some of keys is not set in `os.environ`? Should we fail?

Reply:
dict comprehension - good point

not sure what error handling can be here because we call get_envs_settings only to provide data that was set in _set_environ to xdist workers